content/response_integrations/third_party/torq/release_notes.yaml

```
@@ -0,0 +1,28 @@
+releases:
+  - description: "NEW! Torq integration."
+    integration_version: 1.0
+    item_name: Torq
+    item_type: Integration
+    publish_time: "2025-10-17"
+    ticket_number: ""
+
+  - description: "Ping action."
+    integration_version: 1.0
+    item_name: Ping
+    item_type: Action
+    publish_time: "2025-10-17"
+    ticket_number: ""
+
+  - description: "Create Case in Torq action (supports deduplication and correlation keys)."
+    integration_version: 1.0
+    item_name: Create Case in Torq
+    item_type: Action
+    publish_time: "2025-10-17"
+    ticket_number: ""
+
+  - description: "Run Workflow action (accepts workflow_input; optional include alert payload)."
+    integration_version: 1.0
+    item_name: Run Workflow
+    item_type: Action
+    publish_time: "2025-10-17"
+    ticket_number: ""
```

There's no need for the `releases` key

```suggestion
-   description: "NEW! Torq integration."
    integration_version: 1.0
    item_name: Torq
    item_type: Integration
    publish_time: "2025-10-17"
    ticket_number: ""

-   description: "Ping action."
    integration_version: 1.0
    item_name: Ping
    item_type: Action
    publish_time: "2025-10-17"
    ticket_number: ""

-   description: "Create Case in Torq action (supports deduplication and correlation keys)."
    integration_version: 1.0
    item_name: Create Case in Torq
    item_type: Action
    publish_time: "2025-10-17"
    ticket_number: ""

-   description: "Run Workflow action (accepts workflow_input; optional include alert payload)."
    integration_version: 1.0
    item_name: Run Workflow
    item_type: Action
    publish_time: "2025-10-17"
    ticket_number: ""
```